src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx

```diff
@@ -78,27 +185,56 @@ describe('DetailsViewCommandBar', () => {
             id: 5,
             title: thePageTitle,
             isClosed: false,
+            url: thePageUrl,
             detailsViewId: undefined,
         } as PersistedTabInfo;
+        assessmentStoreData.assessmentNavState = {
+            selectedTestType: 2,
+            selectedTestSubview: 'landmark-roles',
+        };
+        rightPanelConfiguration = {
+            startOverContextMenuKeyOptions: {
+                showTest: true,
+            },
+        } as DetailsRightPanelConfiguration;
     });
 
-    function getProps(): DetailsViewCommandBarProps {
+    function getProps(switcherNavButtonOverrides?: string[]): DetailsViewCommandBarProps {
         const CommandBarStub: ReactFCWithDisplayName<DetailsViewCommandBarProps> =
             NamedFC<DetailsViewCommandBarProps>('test', _ => null);
         const LeftNavStub: ReactFCWithDisplayName<LeftNavProps> = NamedFC<LeftNavProps>(
             'test',
             _ => null,
         );
+
+        const actualSwitcherNavConfiguration = GetDetailsSwitcherNavConfiguration({
+            selectedDetailsViewPivot: DetailsViewPivotType.assessment,
+        });
```

Comment:
we need this function here ? we just need to pass pviot view assessment correct?

Reply:
This is done by Vikash this is fine.